### DISCUSSION TITLE
Make monitor remember last setting on wake

### DISCUSSION THREAD

**Discussion by JimmysDev** [MonitorControl/MonitorControl#1412] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues

### Is your feature request related to a problem? Please describe

Hi there,

Thanks so much for developing this app guys, it's a godsend for me.

The one issue I have is that when my monitor (LG 29WL500, 34" 1080p ultrawide) wakes up, it resets to full brightness. My old Samsung monitor (SyncMaster T240?) also did this.

### Describe the solution you'd like

It would be nice if MonitorControl could detect when the monitor is waking from sleep and restore the previous setting.

### Describe alternatives you've considered

I did find a thread where BetterDisplay was mentioned, which I have now installed. But it would be nice to see this feature in MonitorControl because it seems like quite a fundamental thing.

### Anything else?

Thanks,
Jimmy

**Comment by waydabber** [comment]
If this is DDC birghtness then you could try changing this setting like this: 

<img width="842" alt="Screenshot 2023-08-10 at 10 25 26" src="https://github.com/MonitorControl/MonitorControl/assets/37590873/cf5632a6-4f1d-41a6-b652-e14dd82f1100">

**Comment by daprice** [comment]
Is there a way to get it to reapply the values every time the *monitor* wakes from display sleep or gets plugged in? That setting only seems to apply when the *mac* wakes from system sleep.
